### PR TITLE
Add Lists services

### PR DESCRIPTION
**Pull request by vakuum099**
lists creation;
add contacts from Contacts into the list;
get a list of contacts by listId.

### CODE REVIEW

File: src/contact/contact.service.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import { WorkWithDB } from '../interfaces';
+import { sanitizeRawArray } from '../utils/utils';
+import { GetContactFromDB, IContact } from './contact.interfaces';
+import { Contact } from './contact.model';
+
+export const createNewContact: WorkWithDB = async (req) => {
+  const { name, email } = JSON.parse(req.body.contact) as IContact;
+  try {
+    const res = await Contact.create({
+      uuid: uuidv4(),
```
Comment (CatDidenko) [inline src/contact/contact.service.ts]: model should auto generate this value

File: src/contact/contact.service.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    });
+    return addedData201;
+  } catch (err) {
+    return internalServerError500;
+  }
+};
+
+export const updateExistedContact: WorkWithDB = async (req) => {
+  const uuid = req.params.contactId;
+  const { name, email } = JSON.parse(req.body.contact) as { name: string; email: string };
```
Comment (CatDidenko) [inline src/contact/contact.service.ts]: Why contact info is a string?

File: src/contact/contact.service.ts [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        email,
+      },
+      {
+        where: {
+          uuid,
+        },
+      }
+    );
+    return OK200;
+  } catch (err) {
```
Comment (CatDidenko) [inline src/contact/contact.service.ts]: Why you expect error parameter if you don't use it in the response?

File: src/list/list.service.ts [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+import { ContactsList } from '../contactsList/contactsList.model';
+import { WorkWithDB } from '../interfaces';
+import { showData } from '../utils/utils';
+import { List } from './list.model';
+
+export const createNewList: WorkWithDB = async (req) => {
+  const { name } = JSON.parse(req.body.list) as { name: string };
+  try {
+    const res = await List.create({
+      uuid: uuidv4(),
```
Comment (CatDidenko) [inline src/list/list.service.ts]: This value should be auto generate by Model

File: src/list/list.service.ts [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+    showData(answer);
+    return OK200;
+  } catch (err) {
+    return internalServerError500;
+  }
+};
+
+export const addNewContactToList: WorkWithDB = async (req) => {
+  const listId: string = req.params.listId;
+  const { contactId } = JSON.parse(req.body.contactId) as { contactId: string };
```
Comment (CatDidenko) [inline src/list/list.service.ts]: Why so difficult?

**Review by CatDidenko — CHANGES_REQUESTED** [review]

File: src/list/list.service.ts [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+};
+
+export const getContactsOfList: WorkWithDB = async (req) => {
+  const listId: string = req.params.listId;
+  try {
+    const res = await ContactsList.findAll({
+      attributes: ['contactId'],
+      where: { listId },
+    });
+    const answer = await getListOfContacts(res);
```
Comment (CatDidenko) [inline src/list/list.service.ts]: The task was to create relations between List model and Contacts model. You should use https://sequelize.org/master/manual/assocs.html

**Review by CatDidenko — CHANGES_REQUESTED** [review]

File: src/contact/contact.service.ts (hunk 1 above)
Comment (vakuum099) [reply src/contact/contact.service.ts]: Changed

File: src/list/list.service.ts (hunk 4 above)
Comment (vakuum099) [reply src/list/list.service.ts]: Changed

File: src/list/list.service.ts (hunk 5 above)
Comment (vakuum099) [reply src/list/list.service.ts]: changed

File: src/contact/contact.service.ts (hunk 3 above)
Comment (vakuum099) [reply src/contact/contact.service.ts]: changed

File: src/contact/contact.service.ts (hunk 2 above)
Comment (vakuum099) [reply src/contact/contact.service.ts]: Changed

File: src/list/list.service.ts (hunk 6 above)
Comment (vakuum099) [reply src/list/list.service.ts]: Changed. Utilized features of sequelize.

File: src/sequelize.ts [hunk 7]
```diff
@@ -0,0 +1,6 @@
+import { Sequelize } from 'sequelize-typescript';
+
+export const sequelize = new Sequelize('listsdb', 'root', 'M8NageUdWMEZHXn', {
```
Comment (CatDidenko) [inline src/sequelize.ts]: Password in config!
Password should be in the env variables!

**Review by CatDidenko — APPROVED** [review]

**Pull request merged**